packages/core/src/RenderPipeline/batcher/BatcherManager.ts

```diff
@@ -0,0 +1,44 @@
+import { Engine } from "../../Engine";
+import { RenderContext } from "../RenderContext";
+import { RenderData } from "../RenderData";
+import { RenderElement } from "../RenderElement";
+import { RenderDataUsage } from "../enums/RenderDataUsage";
+import { Batcher2D } from "./Batcher2D";
+
+export class BatcherManager {
```

Comment:
What is the value of this class?

Reply:
future, i want handle all batch logic in batch function